ISLE/xbox_one_series/xbox_controller_map.h

```diff
@@ -0,0 +1,30 @@
+#pragma once
+
+#define SDL_XBOX_BUTTON_A 0
+#define SDL_XBOX_BUTTON_B 1
+#define SDL_XBOX_BUTTON_X 2
+#define SDL_XBOX_BUTTON_Y 3
+#define SDL_XBOX_BUTTON_LEFT_BUMPER 4
+#define SDL_XBOX_BUTTON_RIGHT_BUMPER 5
+#define SDL_XBOX_BUTTON_BACK 6
+#define SDL_XBOX_BUTTON_START 7
+#define SDL_XBOX_BUTTON_LEFT_THUMB 8
+#define SDL_XBOX_BUTTON_RIGHT_THUMB 9
+#define SDL_XBOX_BUTTON_DPAD_UP 10
+#define SDL_XBOX_BUTTON_DPAD_DOWN 11
+#define SDL_XBOX_BUTTON_DPAD_LEFT 12
+#define SDL_XBOX_BUTTON_DPAD_RIGHT 13
+
+#define SDL_XBOX_AXIS_LEFT_Y 0
+#define SDL_XBOX_AXIS_LEFT_X 1
```

Comment:
What's preventing you from using the normal `SDL_GAMEPAD_AXIS_LEFTX`?

Reply:
Also please don't pollute the `SDL_` namespace.